.github/workflows/storybook.yml

```diff
@@ -43,4 +43,5 @@ jobs:
           clean-exclude: |
             pr
             v3
+            mgt-chat
```

Comment:
Comment to block this PR until #2752 is merged and the deployed app at https://mgt.dev/next/mgt-chat has been validated